Octokit/Models/Response/PullRequestFile.cs

```diff
@@ -34,8 +35,10 @@ public PullRequestFile(string sha, string fileName, string status, int additions
         public string RawUrl { get; protected set; }
         public string ContentsUrl { get; protected set; }
         public string Patch { get; protected set; }
+        [Parameter(Key = "previous_filename")]
+        public string PreviousFileName { get; protected set; }
 
-        internal string DebuggerDisplay
+    internal string DebuggerDisplay
```

Comment:
Something happened to whitespace/alignment here?

Reply:
Ah, sorry about that.